### PR TITLE
Add manifest entries for `Add-Exports`

### PR DESCRIPTION
**Pull request by carterkozak**

### CODE REVIEW

File: goethe/src/main/java/com/palantir/goethe/DirectFormatterFacade.java [hunk 1]
```diff
@@ -59,7 +58,7 @@ private static String generateMessage(
                         .append(lines.get(formatterDiagnostic.line() - 1))
                         .append('\n')
                         // Offset by two to convert from one-indexed to zero indexed values, and account for the caret.
-                        .append(Strings.repeat(" ", Math.max(0, formatterDiagnostic.column() - 2)))
+                        .append(" ".repeat(Math.max(0, formatterDiagnostic.column() - 2)))
```
Comment (carterkozak) [inline goethe/src/main/java/com/palantir/goethe/DirectFormatterFacade.java:61]: error-prone suggestion, drive-by

**Review by fawind — APPROVED** [review]
